test/Query.test.ts

```diff
@@ -0,0 +1,69 @@
+import { gql } from "graphql-tag";
+import { ApolloServer } from "apollo-server-express";
+import { resolvers } from "../src/resolvers";
+import fs from "fs";
+const { expect } = require("chai");
```

Comment:
Your test runner is Jest so you don't need to import assertion related functions. Assume it's there.